control/content/controllers/content.item.controller.js

```diff
@@ -478,6 +478,21 @@
             },
             !isNewItemInserted
           );
+          PluginEvents.register(
```

Comment:
I can see that in the ticket we have one more request: Coupon name- opened

Did we forget to put it or do we have it already in the code? If we do have, just ignore my comment.